language/EN.json

```diff
@@ -423,8 +423,8 @@
         },
         "webss": {
             "SS_DESC": "Takes a screenshot from the page in the given link.",
-            "LİNK": "You must enter link!"
-            "SPDF_DESC": "Converts a Site into PDF"
+            "LİNK": "You must enter link!",
+            "SPDF_DESC": "Converts a Site into PDF",
```

Comment:
The end of the strings, you don't need to add **,**
`"SPDF_DESC": "Converts a Site into PDF"`

Reply:
This is okay na? What is the problem I have added , at the end na 💔😟

Reply:
Check line 427. 
Corrupted Code: `"SPDF_DESC": "Converts a Site into PDF",`
Envisaged Code: `"SPDF_DESC": "Converts a Site into PDF"`

Reply:
Okay Now?! 😅